blocks/footer-links/footer-links.css

```diff
@@ -0,0 +1,129 @@
+/* mobile first */
+.footer-links {
+    margin: 0 auto;
+    padding: 24px 27px;
+    background-color: #2b6fb1;
+    display: flex;
+    justify-content: center;
+
+    .mobile-heading {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        width: 100%;
+    }
+
+    .expand-icon,
+    .collapse-icon {
```

Comment:
you could also just use a `.open` additional class that is added when it's open. that way you don't have to swap the css.